core/src/main/scala/io/odin/formatter/Formatter.scala

```diff
@@ -67,14 +65,43 @@ object Formatter {
       builder.toString()
     }
 
+  /**
+    * Formats timestamp using the following format: yyyy-MM-ddTHH:mm:ss,SSS
+    */
+  def formatTimestamp(timestamp: Long): String = {
+    val date = timestamp.t
+    p"${date.F}T${date.T},${date.milliOfSecond}"
+  }
+
+  /**
+    * The result differs depending on the format:
+    *
+    * [[PositionFormat.Full]] - prints full position
+    * 'io.odin.formatter.Formatter formatPosition:75' formatted as 'io.odin.formatter.Formatter formatPosition:75'
+    *
+    * [[PositionFormat.AbbreviatePackage]] - prints abbreviated package and full enclosing
+    * 'io.odin.formatter.Formatter formatPosition:75' formatted as 'i.o.f.Formatter formatPosition:75'
+    */
+  def formatPosition(position: Position, format: PositionFormat): String = {
+    val lineNumber = if (position.line >= 0) p":${position.line}" else ""
+
+    val enclosure = format match {
+      case PositionFormat.Full              => position.enclosureName
+      case PositionFormat.AbbreviatePackage => abbreviate(position.enclosureName)
+
+    }
+
+    p"$enclosure$lineNumber"
+  }
+
   /**
     * Default Throwable printer is twice as slow. This method was borrowed from scribe library.
     *
     * The result differs depending on the format:
-    * `ThrowableFormat.Depth.Full` - prints all elements of a stack trace
-    * `ThrowableFormat.Depth.Fixed` - prints N elements of a stack trace
-    * `ThrowableFormat.Indent.NoIndent` - prints a stack trace without indentation
-    * `ThrowableFormat.Indent.Fixed` - prints a stack trace prepending every line with N spaces
+    * [[ThrowableFormat.Depth.Full]] - prints all elements of a stack trace
```

Comment:
`[[]]` brackets break the `sbt doc` (and therefore the release) unless it contains the absolute path with package, such as
`io.odin.formatter.options.ThrowableFormat.Depth.Full`

You can try it locally using `sbt doc`